commands/General/Voice/play.js

```diff
@@ -110,7 +109,7 @@ module.exports = class extends Command {
       return msg.sendError('NOT_SAME_CHANNEL');
     }
     while (tracks.length > 0) {
-      // console.log('added song:', tracks[0].info.title);
```

Comment:
Same here with unused code thats commented out.

Reply:
this is a comment about how to access the track data in the futue 🤔